translations/en.json

```diff
@@ -276,7 +276,7 @@
         "draft_folder": "[DRAFT]",
         "duplicate_messages": "Duplicate Messages",
         "edit": "Edit",
-        "edit_access": "Edit access",
+        "edit_access": "Edit {{name}}'s access",
```

Comment:
this is wrong

Reply:
Automatic string generation issue. I fixed it